packages/web-components/fast-components/src/skeleton/skeleton.definitions.ts

```diff
@@ -0,0 +1,48 @@
+import { WebComponentDefinition } from "@microsoft/fast-tooling/dist/data-utilities/web-component";
+import { DataType } from "@microsoft/fast-tooling";
+
+export const fastSkeletonDefinitions: WebComponentDefinition = {
+    version: 1,
+    tags: [
+        {
+            name: "fluent-skeleton",
+            description: "The Fluent skeleton element",
+            attributes: [
+                {
+                    name: "fill",
+                    description: "The fill attribute",
+                    type: DataType.string,
+                    default: undefined,
+                    required: false,
+                },
+                {
+                    name: "shape",
+                    description: "The shape attribute",
+                    type: DataType.string,
+                    default: "circle",
+                    required: false,
+                },
+                {
+                    name: "pattern",
```

Comment:
The `pattern` attribute already exists on some HTML elements for regular expression pattern matching (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern).

Reply:
Would there be a collision here? It's a valid attribute but I'd be curious of @EisenbergEffect's thoughts on if we should mind existing attributes. If this were the text field or input, I think we for sure would have an issue. I'm unsure if this is an issue. Confusion could be a concern here, but I think we're going to end up with less-than-ideal names if we avoid any attrribute any HTML element has already used. Just some thoughts.

Reply:
I don't think it's an issue but we could choose another word if we felt that would help avoid confusion. A few synonyms based on quick search: design · decoration · motif · marking · ornament · ornamentation · device · figure